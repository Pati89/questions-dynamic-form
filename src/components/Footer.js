import styles from "../styles/components/Footer.module.css";

const Footer = () => {
  return (
    <footer className={styles.footer} data-testid="footer">
      <p>
        Developed by <b>Patrycja Banaszczyk</b>
      </p>
    </footer>
  );
};

export default Footer;
