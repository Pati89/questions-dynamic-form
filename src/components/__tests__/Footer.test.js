import Footer from "../Footer";
import { render, screen } from "@testing-library/react";

describe("Footer", () => {
  test("should render a Footer", () => {
    render(<Footer />);

    expect(screen.getByTestId("footer")).toBeTruthy();
    expect(screen.getByText("Developed by")).toBeTruthy();
    expect(screen.getByText("Patrycja Banaszczyk")).toBeTruthy();
  });

  test("should render a Footer match snapshot", () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
