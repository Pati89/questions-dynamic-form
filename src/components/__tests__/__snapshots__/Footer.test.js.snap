// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render a Footer match snapshot 1`] = `
<div>
  <footer
    class="footer"
    data-testid="footer"
  >
    <p>
      Developed by 
      <b>
        Patrycja Banaszczyk
      </b>
    </p>
  </footer>
</div>
`;
